cmd/plugin/cli/tableWriter/table.go

```diff
@@ -0,0 +1,78 @@
+package tableWriter
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+)
+
```

Comment:
This logic seems to be specific to info and status commands. What do you think to move it to the corresponding packages?